lib/recurly/transaction.php

```diff
@@ -2,6 +2,18 @@
 
 class Recurly_Transaction extends Recurly_Resource
 {
+
+  protected static $_writeableAttributes;
+  protected static $_nestedAttributes;
+
+  public static function init()
+  {
+    Recurly_Transaction::$_writeableAttributes = array(
+      'account','amount_in_cents','currency','description','accounting_code','payment_method','collected_at'
```

Comment:
The pattern has changed for doing these, we should just have a simple method that returns an array directly.